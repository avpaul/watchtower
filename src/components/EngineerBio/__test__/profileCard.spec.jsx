import React from 'react';
import { shallow } from 'enzyme';
import ProfileCard from '../profileCard';

describe('tests the Profile card', () => {
  const props = {
    firstName: 'Cristian',
    lastName: 'Bartell',
    picture: 'https://'
  };

  it('renders correctly', () => {
    const wrapper = shallow(<ProfileCard {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
