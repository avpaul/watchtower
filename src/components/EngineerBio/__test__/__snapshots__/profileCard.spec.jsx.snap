// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests the Profile card renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ProfileCard
    firstName="Cristian"
    lastName="Bartell"
    picture="https://"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className=" profile_card_details text-center"
>
        <img
                alt=""
                src="https://"
        />
        <h3>
                Cristian
                 
                Bartell
        </h3>
        <p>
                D1 Engineer
        </p>
        <h4>
                Cadre Program
        </h4>
</div>,
      "className": "profile_card",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt=""
            src="https://"
/>,
          <h3>
            Cristian
             
            Bartell
</h3>,
          <p>
            D1 Engineer
</p>,
          <h4>
            Cadre Program
</h4>,
        ],
        "className": " profile_card_details text-center",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "",
            "src": "https://",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Cristian",
              " ",
              "Bartell",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Cristian",
            " ",
            "Bartell",
          ],
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "D1 Engineer",
          },
          "ref": null,
          "rendered": "D1 Engineer",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Cadre Program",
          },
          "ref": null,
          "rendered": "Cadre Program",
          "type": "h4",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className=" profile_card_details text-center"
>
          <img
                    alt=""
                    src="https://"
          />
          <h3>
                    Cristian
                     
                    Bartell
          </h3>
          <p>
                    D1 Engineer
          </p>
          <h4>
                    Cadre Program
          </h4>
</div>,
        "className": "profile_card",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt=""
              src="https://"
/>,
            <h3>
              Cristian
               
              Bartell
</h3>,
            <p>
              D1 Engineer
</p>,
            <h4>
              Cadre Program
</h4>,
          ],
          "className": " profile_card_details text-center",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "",
              "src": "https://",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Cristian",
                " ",
                "Bartell",
              ],
            },
            "ref": null,
            "rendered": Array [
              "Cristian",
              " ",
              "Bartell",
            ],
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "D1 Engineer",
            },
            "ref": null,
            "rendered": "D1 Engineer",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Cadre Program",
            },
            "ref": null,
            "rendered": "Cadre Program",
            "type": "h4",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
